ts/Core/Axis/Axis.ts

```diff
@@ -989,42 +989,69 @@ class Axis {
             return x;
         }
```

Comment:
Now, when looking at it, the body of `between` function looks a lot like what we implemented. Could you maybe investigate, why didn't it work correctly?